.github/workflows/kind-e2e.yaml

```diff
@@ -0,0 +1,156 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+name: KinD tests
```

Comment:
Can we make this workflow name explicitly? e.g. `E2E test on KinD`

Reply:
Done